books/free-programming-books-langs.md

```diff
@@ -1830,6 +1830,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 #### Drupal
 
+* [Drupal at your Fingertips](https://selwynpolit.github.io/d9book/) - Selwyn Polit and Drupal Community Contributors
```

Comment:
```suggestion
* [Drupal at your Fingertips](https://selwynpolit.github.io/d9book/) - Selwyn Polit, Drupal Community Contributors
```

Great addition!

Reply:
@eshellman Any idea how to get this PR merged?

Reply:
commit the suggestion!